client/src/components/showMeMore/Observations.jsx

```diff
@@ -64,7 +64,7 @@ const Observations = props => {
       {(detailData.length ? detailData : observations)
         .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
         .map(observation => (
-          <ExpansionPanel className={classes.expansionPanel} key={`${observation.date}`}>
+          <ExpansionPanel className={classes.expansionPanel} defaultExpanded={detailData.length === 1}>
```

Comment:
I think you may need to rebase off the most current master, as this is a change that's already in master 🤔

Reply:
I think `defaultExpanded` was already there and @fallon7284 added `key`. This should probably be the line:

```jsx
<ExpansionPanel className={classes.expansionPanel} defaultExpanded={detailData.length === 1} key={`${observation.date}`}>
```

Reply:
I didn't touch this, I just merged master the master branch.

Reply:
You added `key` in [this commit](https://github.com/COVID-19-electronic-health-system/Corona-tracker/pull/696/commits/0c93377a237a92956387d5610066755704709ba8#diff-5ec3b63494ab864e6de64614c2785e41R67) and someone else added `defaultExpanded` in [this commit](https://github.com/COVID-19-electronic-health-system/Corona-tracker/commit/8b59b2c75bfc54cdec76df926e28fe79fb24b7f8#diff-5ec3b63494ab864e6de64614c2785e41R66)

When merging, both attributes should be kept, so the line should be:

```jsx
<ExpansionPanel className={classes.expansionPanel} defaultExpanded={detailData.length === 1} key={`${observation.date}`}>
```

Reply:
I see.  I don't remember doing it.  Probably did it because I was seeing an error from React.  Fix is coming.